docs/product/stats/index.mdx

```diff
@@ -23,9 +14,82 @@ With the dropdowns at the top of the page, you can set whether the page displays
+
+### Accepted
+
+Events and attachments that were successfully processed and stored.
+
+### Filtered
+
+Filtered events and attachments in Sentry are intentionally excluded based on predefined criteria to reduce noise and prioritize relevant data. The following reasons are currently defined:
+
+- **Browser Extensions**: Filtered by browser extension.
+- **Error Message**: Filtered based on specific error messages.
+- **Filtered Transactions**: Filtered because it was a call to a filtered transaction.
+- **Invalid CSP**: Filtered due to an invalid Content Security Policy (CSP) policy.
+- **Ip Address**: Filtered by IP address.
+- **Legacy Browsers**: Filtered by certain legacy versions of browsers that are known to cause problems.
+- **Localhost**: Filtered due to localhost restriction.
+- **Others**: Filtered due to other generic filtering rules not specified above.
+- **Release Version**: Filtered by release name (version).
+- **Web Crawlers**: Identified as a known web crawler.
+
+For more details, please consult the [Inbound Filters](/concepts/data-management/filtering/) documentation.
+
+### Rate Limited
+Events and attachments discarded due to rate limits or quota. The following reasons are currently defined:
+
+- **Abuse**: Activated when Sentry is misused.
+- **Cardinality Limited**: Exceeded internally defined cardinality limit.
+- **Rate Limited**: Exceeded allowed event rate.
+- **Spike Protection**: Activated to protect from any sudden quota-depleting spike. See [Spike Protection](/pricing/quotas/spike-protection/) for more information.
+
+### Invalid
+Events and attachments discarded by the SDK. The following reasons are currently defined:
+
+- **Cors**: A request from an unallowed origin was sent to Sentry. Check project settings to configure allowed domains.
+- **Duplicate**: An event with the same id has already been processed for this project. Sentry does not allow duplicate events and only stores the first one.
+- **Empty**: An empty request or a request missing its primary data component was submitted.
+- **Invalid Data**: Data sent by the SDK was invalid and does not meet the basic schema. This likely indicates an SDK implementation bug. Please report a bug if this issue persists.
+- **Invalid Request**: The HTTP submission was invalid or was missing required information. This is likely due to a problem in the SDK implementation. Please report a bug if this issue persists.
+- **Minidump**: The minidump in a minidump upload request was missing or invalid. See the [Minidumps documentation](/platforms/native/guides/minidumps/) for details on how to upload minidumps.
+- **Other**: An internal problem at Sentry prevented regular processing or storage of the submitted information.
```

Comment:
Let's move "other" to the very bottom. 

I'm wondering if it would become more clear if we call this "internal", what do you think?

Reply:
Not "invalid", but "internal"

Reply:
oh sorry....misread it - internal sounds good